### DISCUSSION TITLE
old proposal for label edition

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-androidapp#3959] [Ideas]
![image](https://cloud.githubusercontent.com/assets/1689815/21966197/2f9ea766-db6f-11e6-9469-a0e78c025de8.png)

Either a checkbox like bio, or more probably an editable field with autocomplete

**Comment by asmoth360** [comment]
@teolemon Good idea, it looks good. And when you type something in the input field, it filters the logos ?
